csharp/src/Ice/InputStream.cs

```diff
@@ -622,24 +622,6 @@ public byte readByte()
         }
     }
 
-    /// <summary>
-    /// Extracts an optional byte value from the stream.
-    /// </summary>
-    /// <param name="tag">The numeric tag associated with the value.</param>
-    /// <param name="isset">True if the optional value is present, false otherwise.</param>
-    /// <param name="v">The optional value.</param>
-    public void readByte(int tag, out bool isset, out byte v)
```

Comment:
All the "out bool isset" methods were unused, and there is no advantage in providing them, so I removed them.